hardhat.config.ts

```diff
@@ -8,14 +8,19 @@ import {
 // Hardhat plugins
 import '@nomiclabs/hardhat-ethers'
 import '@nomiclabs/hardhat-waffle'
+import 'hardhat-deploy'
 import '@typechain/hardhat'
 import '@eth-optimism/plugins/hardhat/compiler'
+import './hh'
 
 const config: HardhatUserConfig = {
   networks: {
     hardhat: {
       accounts: DEFAULT_ACCOUNTS_HARDHAT,
       blockGasLimit: RUN_OVM_TEST_GAS * 2,
+      live: false,
+      saveDeployments: false,
+      tags: ['test', 'local'],
     },
```

Comment:
Should probably add new mainnet/rinkeby/kovan etc. in the networks for our actual deployments.